packages/components/scripts/wrapper-generator/UXPinReactWrapperGenerator.ts

```diff
@@ -309,6 +310,14 @@ export class UXPinReactWrapperGenerator extends ReactWrapperGenerator {
         props: { label: 'Some label', description: 'Some description' },
         children: '<DummyImg uxpId="dummy-img" />',
       },
+      'p-link-tile-model-signature': {
+        props: { heading: 'Some heading' },
+        children: [
+          '<DummyImg uxpId="dummy-img" />',
+          '<Link slot="primary" variant="primary" theme="dark" href="#" uxpId="link-primary">Some link</Link>',
```

Comment:
I guess `variant` and `theme` need to be set here to have the correct values in the editor?

Reply:
Correct, its not set automatically in uxpin